### PR TITLE
Fixes and updates

### PR DESCRIPTION
**Pull request by neodesys**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,2 +1,2 @@
-* text=auto eol=lf
 *.bat text eol=crlf
+* text=auto eol=lf
```
Comment (aperezdc) [inline .gitattributes:2]: This seems like an unnecessary change... I guess the idea is to have the lines sorted alphabetically?

**Review by aperezdc — APPROVED** [review]

File: .gitattributes (hunk 1 above)
Comment (neodesys) [reply .gitattributes:2]: * is greedy, so .bat files were just matching the first line and were taking LF as EOL.
I became aware of it when upgrading gradle, it wrote a new gradelw.bat with the wrong EOL.

Comment (aperezdc) [reply .gitattributes:2]: TIL! Thanks for commenting.
